### ISSUE TITLE
Training word embeddings

### ISSUE THREAD

**Issue by stefan-it** [zalandoresearch/flair#181]
Hi :)

I'm not sure how the word embeddings, specified [here](https://github.com/zalandoresearch/flair/blob/master/flair/embeddings.py#L133) were trained, so I would like to ask if you have some hints for training word embeddings for a new language. E.g. what processing steps (special tokenization, what tokenizer) were used, or what exact parameters did you use for training them? I guess `gensim` and its `fasttext` module was used.

It would be awesome if we could integrate such a word embeddings trainer in `flair` :)

Thanks in advance :heart:

**Comment by alanakbik** [comment]
Hi! For regular embeddings, we used the pre-computed embeddings released by [FastText](https://github.com/facebookresearch/fastText/blob/master/pretrained-vectors.md) and [Glove](https://nlp.stanford.edu/projects/glove/). The good thing about FastText especially is that they made embeddings available for a very large set of languages. A downside however is that embedding files tend to be huge, around several GB per language - this causes some practical issues with large model sizes that we are looking at. 

We've looked a bit into using FastText and Glove to train our own embeddings - they explain how to use their software on the respective web pages. It's very easy to train, but generally requires very large collections of data. In our (limited) experiments, we generally found that the choice of word embeddings (i.e. which embeddings we use) does not impact the downstream result too much - I think this is because the learning approach mostly relies on the character LM features and not so much the word embedding features.
